src/metaswitch/clearwater/etcd_tests/etcdtestbase.py

```diff
@@ -35,9 +35,10 @@
 import sys
 import unittest
 from .etcdcluster import EtcdCluster
+from metaswitch.common.logging_config import configure_test_logging
+
```

Comment:
What's your opinion here? I'm going to fix up etcdcluster as it looks weird to me.